frontend/server/src/Controllers/Problem.php

```diff
@@ -4742,6 +4743,36 @@ public static function getProblemListForSmarty(
         ];
     }
 
+    /**
+     *
+     * @return array{smartyProperties: array{payload: ProblemListCollection, title: string}, entrypoint: string}
+     *
+     * @omegaup-request-param string $problem_level_tag
+     */
+    public static function getProblemCollectionDetailsForSmarty(
+        \OmegaUp\Request $r
+    ): array {
+        return [
+            'smartyProperties' => [
+                'payload' => [
+                    'levelTags' => \OmegaUp\Controllers\Tag::getLevelTags(),
+                    'problemsAmount' => \OmegaUp\DAO\Problems::getAmountOfProblems(
+                        $r->ensureString(
+                            'problem_level_tag',
+                            fn (string $problemLevelTag) => \OmegaUp\Validators::stringNonEmpty(
```

Comment:
acá es mejor usar `alias()` para hacer validación semántica.

```suggestion
                            fn (string $problemLevelTag) => \OmegaUp\Validators::alias(
```